src/main/scala/nl.knaw.dans.easy.multideposit/actions/ValidatePreconditions.scala

```diff
@@ -68,15 +69,22 @@ class ValidatePreconditions(ldap: Ldap, ffprobe: FfprobeRunner) extends DebugEnh
     logger.debug("check that the depositor is an active user")
 
     val depositorUserId = deposit.depositorUserId
-    ldap.query(depositorUserId)(attrs => Option(attrs.get("dansState")).exists(_.get().toString == "ACTIVE"))
+    ldap.query(depositorUserId)(validateDepositorUserId(deposit.row, depositorUserId))
       .flatMap {
         case Seq() => InvalidInput(deposit.row, s"depositorUserId '$depositorUserId' is unknown").asLeft
         case Seq(head) => head.asRight
         case _ => ActionError(s"There appear to be multiple users with id '$depositorUserId'").asLeft
       }
-      .flatMap {
-        case true => ().asRight
-        case false => InvalidInput(deposit.row, s"The depositor '$depositorUserId' is not an active user").asLeft
-      }
+      .flatMap(identity)
+  }
+
+  def validateDepositorUserId(row: Int, depositorUserId: DepositorUserId)(attrs: Attributes): FailFast[Unit] = {
+    lazy val existingDepositorId = Option(attrs.get("uid")).exists(attr => attr.get().toString == depositorUserId)
```

Comment:
Perhaps nice layout, but no need to have this first one lazy as it will always be evaluated.